packages/design-system/src/directives/OcTooltip.ts

```diff
@@ -47,14 +47,14 @@ export const destroy = (_tippy) => {
   }
 }
 
-const initOrUpdate = (el, { value = {} }: any, { elm }) => {
+const initOrUpdate = (el, { value = {} }: any) => {
   if (Object.prototype.toString.call(value) !== '[object Object]') {
     value = { content: value }
   }
 
-  if (value.content !== 0 && !value.content) {
-    destroy(elm.tooltip)
-    elm.tooltip = null
+  if ((value.content !== 0 && !value.content) || value.content === '') {
```

Comment:
@fschade do you remember why we had `el` and `elm`?

Reply:
we had this to be sure that no other tooltip instance is already attached to the element.

Reply:
was historically needed for two tippy instances running in parallel (e.g. when having an `OcDrop` instance open and at the same time displaying a tooltip). Seems like it's not needed anymore.